pipeline-template_impl.hpp

```diff
@@ -4,51 +4,50 @@ namespace mosaic::pipeline {
 
 template <typename ValueT>
 void Producer<ValueT>::produce(ValueT v) {
-    std::unique_lock lock(m_box_this->m_mutex);
-
-    if (!m_box_this)
-        throw std::logic_error("Can't produce values without being associated with a pipeline");
-    assert(m_box_this->m_associated_pipeline);
-
-    if (m_consumers.empty())
-        return; // Don't bother producing values if noone listens
-
-    m_buffered.emplace_back(std::move(v));
-    m_box_this->m_associated_pipeline->register_waiting_producer(this);
+  if (!m_box_this)
+      throw std::logic_error("Can't produce values without being associated with a pipeline");
```

Comment:
Koukam ze jsi zacal delat odsazeni dvema mezerami, vysledek je ze to je nekonzistentni :). Ja zacal se ctyrmi, protoze dve mi prijdou trochu uhozene a tohle je jiny projekt nez M51FW. Chtelo by to srovnat na jednu variantu, asi zalezi na tobe kterou.

Reply:
lol... Linux kernel coding style na vas!
https://www.kernel.org/doc/html/v4.11/process/coding-style.html#indentation